packages/theming/atlas/scripts/build.js

```diff
@@ -60,11 +60,23 @@ concurrently(
                   }
               ]
             : []
-    )
+    ),
+    {
+        prefix: "name",
```

Comment:
Are you sure this works? If you want to combine more prefixes you need `{}`

Reply:
That's not necessary because that's default value.